features/calculation_weekly_frequency.feature

```diff
@@ -0,0 +1,26 @@
+Feature:
+  As a enrolee into a Workplace Pension,
+  I would like to calculate pension contribution on my weekly salary.
+
+  Background:
+    Given I am on the YourDetailsPage
+
+    Scenario Outline:
+      Given I enter my age as "<age>"
```

Comment:
http://pdf.th7.cn/down/files/1411/The%20Cucumber%20Book.pdf
So I've found the relevant pages in the cucumber book. 
Page 65 - data tables. - could maybe investigate improving the layout of this
Page 76 - 5.4 Nesting Steps - useful in reference to below 
Page 90 - Incidental details